routes/results/getBenefits.js

```diff
@@ -169,7 +172,31 @@ const getProvincialBenefits = (data) => {
   return data.province ? 'province-' + data.province : false
 }
 
+const getAllBenefits = () => {
+  const benefitList = [];
+
+  // Get a list of all the benefit cards (except provincial)
+  const files = glob.sync("**/*.njk", {
+    cwd: path.join(__dirname, '../../views/benefits'),
```

Comment:
This seems clever, but then one problem we get is we get multiple CERB cards in the "not for you" that look like duplicates (since we have cerb_ei_sickness, cerb_ei_regular, and finally just cerb).

Reply:
we'll just remove the other cerb cards.